RELEASE

```diff
@@ -5,6 +5,75 @@
 /_/  |_/ .___/ .___//____/\___/\__,_/_/ \___/
       /_/   /_/
 
+AppScale version 3.5.0, released March 2018
+Highlights of features/bugs in this release:
+
+ - Add a datastore viewer to the dashboard
+ - Improve lb responsiveness
+ - Transaction groomer
+ - Add GQL query to datastore viewer
+ - Service statistics
+ - Taskqueue service stats
+ - Add push queue stats to Hermes
+ - Don't allow editing entities with datastore viewer
+ - Allow group locks to persist when commits fail
+ - Allow different instance type per role
+ - Handle key inequality filter within property value
+ - Change appengine role to compute
+ - Whitelist inbound service warmup
+ - Run apps without application element in app.yaml
+ - Use prefix for the haproxy app config file
+ - Prevent ProjectGroomer worker from stopping
+ - Stick to the duty cycle length
+ - Capture Net::ReadTimeout when using Net::HTTP
+ - Haproxy consolidate apps
+ - Patch for GAE Issue # 12783 - Conditional import of RAND_egd from _ssl
+ - Don't clear cron jobs during an up
+ - Remove tcp_tw_recycle option
+ - Replace reference to deprecated role
+ - Reduce Cassandra memory if running other services
+ - Allow the dashboard to be updated
+ - Differentiate between project IDs and version keys
+ - Encode entity names when acquiring lock
+ - Add external API server
+ - Rsync appcontroller client
+ - Allow Hermes to check RabbitMQ status
+ - Fix JSP compilation under Java 8
+ - Handle missing WEB-INF directory
+ - Account properly for pending AppServer.
+ - Handle UpdateCron requests
+ - Quote environment variables
+ - Don't wait for taskqueue master to start rabbitmq
+ - Autoscale with N load balancer nodes
+ - Set log after restart
+ - Clear logs when flushing
+ - Add support for Jessie
+ - Remove support for precise
+ - Sync & Async Retrying python decorators in appscale-common
```

Comment:
` - Sync & Async Retrying python decorators in appscale-common` => `Use tornado coroutines when do retries to not waste backoff time`.